docs/business.yaml

```diff
@@ -1564,15 +1564,604 @@ paths:
+                        business: 
+                          natureOfBusiness: sample business
+                        businessType: SP
+                        offices: 
+                          businessOffice: 
+                            deliveryAddress: 
+                              streetAddress: delivery_address - address line one
+                              addressCity: delivery_address city
+                              addressCountry: Canada
+                              postalCode: H0H0H0
+                              addressRegion: BC
+                            mailingAddress: 
+                              streetAddress: mailing_address - address line one
+                              addressCity: mailing_address city
+                              addressCountry: Canada
+                              postalCode: H0H0H0
+                              addressRegion: BC
+                        contactPoint: 
+                          email: no_one@never.get
+                        startDate: '2024-07-26'
+                        nameRequest: 
+                          nrNumber: NR 8332083
+                          legalName: KD API SPEC SP
+                          legalType: SP
+                        parties:
+                          - mailingAddress: 
+                              streetAddress: mailing_address - address line one
+                              streetAddressAdditional: 
+                              addressCity: mailing_address city
+                              addressCountry: CA
+                              postalCode: H0H0H0
+                              addressRegion: BC
+                            deliveryAddress: 
+                              streetAddress: delivery_address - address line one
+                              streetAddressAdditional: 
+                              addressCity: delivery_address city
+                              addressCountry: CA
+                              postalCode: H0H0H0
+                              addressRegion: BC
+                            officer: 
+                              firstName: Joe
+                              lastName: Swanson
+                              middleName: P
+                              email: joe@email.com
+                              organizationName: 
+                              partyType: person
+                            roles:
+                              - appointmentDate: '2024-07-26'
+                                roleType: Completing Party
+                              - appointmentDate: '2024-07-26'
+                                roleType: Proprietor
+        '401':
+          description: Similar to 403 Forbidden, but specifically for use when authentication is required and has failed or has not yet been provided. The user does not have valid authentication credentials for the target resource.
+          content:
+            application/json:
+              examples:
```

Comment:
Would you mind to add these 2 failed examples to the Postman collection? May help users to understand different situations.

Reply:
When a user is Not authorized to Registration or amalgamation for given NR or account Id. I can add `not-authorized-to-submit` to the name
![image](https://github.com/user-attachments/assets/b0258195-1c63-4568-88b2-804b9547858d)

Reply:
or maybe mention the root cause is due to use an accountID or  NR that is not authorized to the user? I was just thinking about how to make it easier to understand.

Reply:
I think this explains the reason well enough

Reply:
> Would you mind to add these 2 failed examples to the Postman collection? May help users to understand different situations.

Added Fail examples to Postman for each type